cmd/aries-agent-mobile/pkg/wrappers/rest/rest.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package rest
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+func makeHTTPRequest(httpClient *http.Client, method, url, token string, _ interface{}) ([]byte, error) {
```

Comment:
Why there is non variable interface  this one in the makeHTTPRequest signature  _ interface{}??

Reply:
that's supposed to be request body. i'll remove it since it's not needed/used yet